app/views/sequences/_form.html.haml

```diff
@@ -29,11 +29,6 @@
           [preview]
         .activity_thumbnail#thumbnail_preview
           %img
-    .field
-      = f.label :external_report_url, t("EXTERNAL_REPORT_URL")
-      = f.text_field :external_report_url
-      %br
-      .hint=t("EXTERNAL_REPORT_WARNING")
```

Comment:
Maybe too OCD of me but shouldn't `EXTERNAL_REPORT_WARNING` be removed from en.yml?

Reply:
Nice catch.

Reply:
Added 7ce35f8 to do this.  Thanks.